homeassistant/components/rfxtrx/sensor.py

```diff
@@ -121,6 +155,9 @@ def __init__(self, device, name, data_type, should_fire_event=False, event=None)
             f"{device.packettype:x}_{device.subtype:x}_{device.id_string}_{data_type}"
         )
 
+        self._device_class = DEVICE_CLASSES.get(data_type, None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
```suggestion
        self._device_class = DEVICE_CLASSES.get(data_type)
```